docs: maintainers (#5198)
EddieHubCommunity/LinkFree: pages/docs/contributing/maintainers.mdx

import DocsLayout from "../../../components/layouts/DocsLayout.js";

## Maintainers

- Only assign issues with the label `status: ready for dev`
- Each person should only be assigned 1 open issue at a time

---

> `avatar` json property is included

> This is no longer used, please leave an inline comment with the code sugestion to remove it - this will allow another maintainer to accept the changes and we can get the Pull Request merged without delay.

Reviewer: ```suggestion
**Note:** This is no longer used, please leave an inline comment with the code sugestion to remove it - this will allow another maintainer to accept the changes and we can get the Pull Request merged without delay.
```

```diff
- "avatar": "http://github.com/SaraJaoude.png",
+
```

If you wish to make the fix yourself, you can merge to a temporary branch and fix, then create a new Pull Request.

> _Note: multiple Pull Requests that need fixing can go into this temporary branch, but please make sure this branch is not open for longer than a few hours to avoid any confusion._

Reviewer: This is a great start @eddiejaoude! 😄 However, the word "this" in "please make sure this branch" implies that there is a specific branch in place, so maybe we can say this instead:
Note: multiple Pull Requests that need fixing can go into this temporary branch, but please make sure the branch is not open for longer than a few hours to avoid any confusion.

Author: Thank you Chrissy! Btw you can click the "suggest" button, which will add the changes in the inline comment and you can amend it there with your suggestions, there are 2 reasons why this is great...
1. I can just click accept 
2. you get a green screen for making the changes/commit

<img width="822" alt="Screenshot 2023-03-07 at 11 54 05" src="https://user-images.githubusercontent.com/624760/223324232-62cb33d1-819a-4564-aacd-05a510f84527.png">

> _Note: multiple Pull Requests that need fixing can go into this temporary branch, but please make sure this branch is not open for longer than a few hours to avoid any confusion._

Reviewer: ```suggestion
_Note: multiple Pull Requests that need fixing can go into a temporary branch, but please make sure it is not open for longer than a few hours to avoid any confusion._
```

---

> If the author used an alternative icon (for example, a `FaDollarSign` for their Instagram).

Please do **not** accept any suggested changes on their behalf, this should **not** block merging the Pull Request.

---

> Author has misspelt their GitHub username (including the incorrect case) and the PR has been open for over 5 days and there are no other changes requested/required

1. Create a temporary branch, update the PR to go to this temporary branch
2. Leave a comment in the original Pull Request explaining you will merge into a temporary branch and make the changes
3. Merge into your temporary branch, make the changes
4. Create a new Pull Request and merge to main

_Note: multiple Pull Requests that need fixing can go into this temporary branch, but please make sure this branch is not open for longer than a few hours to avoid any confusion._

---

> Json package lock fix

---

> Not having any links should not be a blocker to merging the PR (despite the name being LinkFree!)

Please suggest they add links when they are ready and make reference to the Editing Guide in Docs. Then merge the PR.

---

> Conflicts have arisen as the fork is outdated

> In the first instance please ask them to resolve these and you may want to provide a link to this video that explains how: https://youtu.be/eze7Y4Rxwtc

Reviewer: Hi @eddiejaoude! 😄 This is a great start but I found a grammatical error in this sentence. When maintainers see, "In the first instance", they wonder if you're introducing a step. Consider using this method to make it clearer:
* In the first instance**,**please ask them to resolve these and you may want to provide a link to this video that explains how: https://youtu.be/eze7Y4Rxwtc 
I added a comma after "In the first instance" because it's introducing the step.
I hope this helps! 😄

> In the first instance please ask them to resolve these and you may want to provide a link to this video that explains how: https://youtu.be/eze7Y4Rxwtc

Reviewer: ```suggestion
In the first instance, please ask them to resolve this problem. You may want to provide a link to this video that explains how: https://youtu.be/eze7Y4Rxwtc
```

> If there is no activity by the author to resolve this (approx 5 days or more) then consider resolving the conflicts by accepting the changes in the PR

Reviewer: See my comment on line 81.

> If there is no activity by the author to resolve this (approx 5 days or more) then consider resolving the conflicts by accepting the changes in the PR

Reviewer: ```suggestion
If there is no activity by the author resolves this (approx 5 days or more), then consider resolving the conflicts by accepting the changes in the PR
```
